spec/api-web-contents-view-spec.ts

```diff
@@ -17,6 +17,20 @@ describe('WebContentsView', () => {
     new WebContentsView({});
   });
 
+  it('accepts existing webContents object', async () => {
+    const currentWebContentsCount = webContents.getAllWebContents().length;
+
+    const wc = (webContents as typeof ElectronInternal.WebContents).create({ sandbox: true });
+    await wc.loadURL('about:blank');
+
+    const webContentsView = new WebContentsView({
+      webContents: wc
+    });
+
+    expect(webContentsView.webContents).to.eq(wc);
+    expect(webContents.getAllWebContents().length).to.equal(currentWebContentsCount + 1, 'expected only single webcontents to be created');
+  });
+
```

Comment:
Please also test what happens if you create a WebContentsView with a WebContents that's already attached somewhere else (either a BrowserWindow's WebContents or another WebContentsView's WebContents).

Reply:
I've added the check for owner window when creating `WebContentsView`. Please let me know if there is a better way to determine whether `WebContents` is already attached to something.

Reply:
tests are passing, lgtm